include/swift/Reflection/ReflectionContext.h

```diff
@@ -1310,7 +1310,7 @@ class ReflectionContext
 #define TAG(name, value)                                                       \
   case value:                                                                  \
     return std::string(#name);
-#include "../../../stdlib/public/runtime/MetadataAllocatorTags.def"
+#include "runtime/MetadataAllocatorTags.def"
```

Comment:
My concern with `runtime` being alone here is that we also have the `swift/Runtime` directory in the main repository.

Reply:
Yeah, I think that seems like a reasonable change.

The only concern would be if the file actually declared or defined anything, moving it could have an impact on public API, but since it's just an x-macro that gets expanded to something else, I don't think it will break anything.